tasks/irsInstances.ts

```diff
@@ -45,6 +47,23 @@ async function writeIrsCreationTransactionsToGnosisSafeTemplate(
   console.log("Output:\n", output);
 }
 
+async function getLpMarginCap(
+  hre: HardhatRuntimeEnvironment,
+  underlyingToken: IERC20Minimal,
+  stableUnderlying: boolean
+) {
+  // defaults to eth value because it's lower
+  let lpMarginCapUnscaled = getConfig(hre.network.name).irsConfig.lpMarginCap.eth;
+  if(stableUnderlying) {
+    lpMarginCapUnscaled = getConfig(hre.network.name).irsConfig.lpMarginCap.stableCoin;
+  }
+
+  const decimals = await underlyingToken.decimals();
```

Comment:
i think we still can do that for ETH pools, see https://github.com/Voltz-Protocol/voltz-core/blob/main/contracts/rate_oracles/LidoRateOracle.sol#L31 and https://github.com/Voltz-Protocol/voltz-core/blob/main/contracts/rate_oracles/RocketPoolRateOracle.sol#L24

Reply:
hmm, yeah i think that's true since we get the decimals from weth rather than eth which is erc20 and has 18 decimals as well. would be good to test this to reassure ourselves

Reply:
> hmm, yeah i think that's true since we get the decimals from weth rather than eth which is erc20 and has 18 decimals as well. would be good to test this to reassure ourselves

Agree with this.

Reply:
just tested - both 6 (USDC) and 18 decimals